dockerfiles/bases/pingcap-base/Dockerfile

```diff
@@ -2,7 +2,6 @@
 # hub.pingcap.net/bases/pingcap-base:v1.7
 # hub.pingcap.net/bases/pingcap-base:v1
 FROM rockylinux:9.2.20230513
-RUN --mount=from=busybox:1.36.1,source=/bin,target=/busybox/bin \
```

Comment:
It will fail in kaniko building.
Also we should not mix copy in run steps.

Reply:
just for less layers